### DISCUSSION TITLE
Multi-node Deepspeed

### DISCUSSION THREAD

**Discussion by aced125** [Lightning-AI/pytorch-lightning#7005] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi

Is there any tutorial to do perform Deepspeed training on multi-node with lightning? Thanks

cc: @SeanNaren

**Comment by SeanNaren** [comment]
Hey :)

Thanks for PMing me offline! Just to summarize, currently there is no tutorial but it should work the same way as multi-node training for conventional plugins. I'm hoping in the future we'll have a minimal example to show, as we discussed offline :)
